debian/stretch/foreman-installer/changelog

```diff
@@ -0,0 +1,107 @@
+foreman-installer (1.16.0-1) stable; urgency=low
+
+  * Bump changelog to 1.16.0 to match VERSION
+
+ -- Michael Moll <mmoll@mmoll.at>  Thu, 30 Mar 2017 10:54:21 +0200
+
+foreman-installer (1.14.2-1) stable; urgency=low
```

Comment:
Probably also delete these changelog entries for stretch as they weren't shipped, just keep 1.16.0.